jlama.java

```diff
@@ -1,8 +1,9 @@
 ///usr/bin/env jbang "$0" "$@" ; exit $?
-//COMPILE_OPTIONS -source 20
+//JAVA 21+
+//PREVIEW 
```

Comment:
This automatically handles --enable-preview and source targets are aligned. 

You can also do it all manually and explicitly but lets things that can break this way.